docusaurus/README.md

```diff
@@ -0,0 +1,58 @@
+# Documentation and Docusaurus
+
+We use [docusaurus](https://docusaurus.io) for consistent process, in `Airbyte` **no website is generated**.
+Functionally this is a very fancy **linter**
+
+Running the build process will **check for broken links**, please read the output and address
+any broken links that you are able to do.
+
+## Installation
+
+For consistency across other Airbyte projects we use yarn (A Javascript based software package manager)
+
+```bash
+brew install yarn
+
+cd docusaurus
+yarn install
+yarn build
+```
+
+At this point you will see any broken links that docusaurus was able to find.
+
+## Developing Locally
+
+```bash
+yarn start # any changes will automatically be reflected in your browser!
+```
+
+## Making Changes
+
+All the content for docs.airbute.com lives in the `/docs` directory in this repo. All files are markdown. Make changes or add new files, and you should see them in your browser!
```

Comment:
New files will not actually be shown - users must add them to the sidebars.js (and maybe rebuild?  not sure) to get them to show up